### ISSUE TITLE
Problems with call end button

### ISSUE THREAD

**Issue by manishbasnyat182** [Mupati/laravel-video-chat#1]
After End Call CTA, the view re-rendering takes a long time, sometimes freezes up.

**Comment by Mupati** [comment]
Yeah, just look at the end Call method and tweak it for your use case. This is a basic setup to get you started.
